### PR TITLE
Editing indicator layouts

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21425

## Description
To each existing layout - I added the "_5.4.9" suffix.
Each indicator layout is based on the below picture. in cases additional fields existed in Ryan's layout I left them there, if Sources section was missing (which happened to some - I added it).

Should be noted this fails validation! it is unclear if it is because demisto-sdk does not support the layouts or if downloaded jsons from Demisto are miss-formed. The error could be seen in the picture below as well.

## Screenshots
Paste here any images that will help the reviewer
image

validation failure:
image

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by roysagi** [comment]
no docs adding docs approved

File: Layouts/layout-indicatorsDetails-Account.json [hunk 1]
```diff
@@ -1,188 +1,297 @@
...
+		"id": "accountRep",
+		"kind": "indicatorsDetails",
+		"modified": "2020-02-13T16:20:38.30086+02:00",
+		"name": "",
+		"prevKind": "indicatorsDetails",
+		"prevTypeId": "accountRep",
+		"shouldCommit": false,
+		"sortValues": null,
+		"system": false,
+		"tabs": [
```
Comment (avidan-H) [inline Layouts/layout-indicatorsDetails-Account.json]: It looks like this file is using tabs... and may be misformatted towards the bottom? It's hard to say - but please take a look.

File: Layouts/layout-indicatorsDetails-Host.json [hunk 2]
```diff
@@ -1,226 +1,326 @@
...
-    "version": -1,
-    "id": "hostRep"
-  },
-  "TypeName": "Host",
-  "version": -1,
-  "fromVersion": "5.0.0"
+	"TypeName": "hostRep",
+	"kind": "indicatorsDetails",
+	"layout": {
+		"TypeName": "",
```
Comment (avidan-H) [inline Layouts/layout-indicatorsDetails-Host.json]: Tabs... ichsa.

File: Layouts/layout-indicatorsDetails-Account.json (hunk 1 above)
Comment (roysagi) [reply Layouts/layout-indicatorsDetails-Account.json]: it seems this is something in how git parses this file... locally it looks fine

**Comment by roysagi** [comment]
@avidan-H - The tab atrocity is most likely a GitHub issue of how it parses this fie - locally it seems fine... I'm not sure I can change this...

File: Layouts/layout-indicatorsDetails-Account_5.4.9.json [hunk 3]
```diff
@@ -0,0 +1,246 @@
+{
```
Comment (avidan-H) [inline Layouts/layout-indicatorsDetails-Account_5.4.9.json]: You're still using tabs in this file.

**Comment by roysagi** [comment]
> Please fix the tabs in `Layouts/layout-indicatorsDetails-Account_5.4.9.json`.
> 
> Also, some of your `5.4.9` files are using 2-space tabs and all of the files you fixed up are using 4-space tabs... A bit nitpicky, I know, but it makes sense to be uniform across files. If you have no koach for that tweak, then I'm ok with you merging without.

I'll just reformat everything...

**Comment by yuvalbenshalom** [comment]
force merging because the sdk dose not know yet how to process the addition of a new fromversion field and a new file with the old version, so it identifies it as a break of BC.

**Pull request merged**